mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/mail/task/SolveMailboxInconsistenciesService.java

```diff
@@ -67,8 +69,8 @@
         }
 
         @Override
-        public Mono<Boolean> isStillPertinent(Context context, CassandraMailboxDAO mailboxDAO, CassandraMailboxPathV2DAO pathV2DAO) {
-            return Mono.just(INCONSISTENCY_STILL_PRESENT);
+        public Mono<Inconsistency> validate(CassandraMailboxDAO mailboxDAO, CassandraMailboxPathV2DAO pathV2DAO) {
```

Comment:
filter

Reply:
I'm not expecting a filter method to mutate something into something else.

I propose `actualize`

Reply:
LGTM